tests/test_worker.py

```diff
@@ -60,6 +60,19 @@ def test_worker_stop(hq_env: HqEnv):
     assert "worker not found" in response
 
 
+def test_worker_stop_some(hq_env: HqEnv):
+    hq_env.start_server()
+    processes = [hq_env.start_worker() for i in range(3)]
+
+    wait_for_worker_state(hq_env, [1, 2, 3], ["RUNNING" for i in range(3)])
```

Comment:
`["RUNNING" for i in range(3)]` can be written more succintly as `["RUNNING"] * 3`.
But here it's enough to just specify `"RUNNING"` as the last argument, it's converted internally to a set anyway.